File: android/src/main/java/com/getcapacitor/community/classes/results/IsStartedResult.java
package com.getcapacitor.community.classes.results;

import com.getcapacitor.JSObject;
import com.getcapacitor.community.interfaces.Result;

public class IsStartedResult implements Result {

    private final Boolean isStarted;

    public IsStartedResult(Boolean isStarted) {
        this.isStarted = isStarted;
    }

    @Override
    public JSObject toJSObject() {
        JSObject result = new JSObject();

        if (isStarted != null) result.put("isStarted", isStarted);

        return result;
    }
}

File: ios/Sources/BridgefyPlugin/Classes/Results/IsStartedResult.swift
import Foundation
import Capacitor

@objc public class IsStartedResult: NSObject, Result {
    let isStarted: Bool

    init(_ isStarted: Bool) {
        self.isStarted = isStarted
    }

    public func toJSObject() -> AnyObject {
        var result = JSObject()

        result["isStarted"] = isStarted

        return result as AnyObject
    }
}
